docs/build/dapp-staking/indexer.md

```diff
@@ -0,0 +1,112 @@
+# The API and the Indexer
+
+The API is accessible from a [Swagger User Interface](https://api.astar.network/) to test the endpoints out.
+
+|        |                                                                |
+| ------ | -------------------------------------------------------------- |
+| UI     | https://api.astar.network/                                     |
+| API    | https://api.astar.network/api/v3/{network}/dapps-staking/{...} |
+| Github | https://github.com/AstarNetwork/astar-token-api                |
+
+## Available endpoints
+
+| Data                                                   | Endpoint, start with `/api/v3/{network}/dapps-staking` |
+| ------------------------------------------------------ | ------------------------------------------------------ |
+| List of dapps registed for staking                     | `/chaindapps`                                          |
+| TVL for a given network and period                     | `/tvl/{period}`                                        |
+| List of stakers per dapp with total stake              | `/stakerslist/{contractAddress}`                               |
+| Stakers count for a given network for a dapp by period | `/stakerscount/{contractAddress}/{period}`             |
+| Total stakers count for a given network and period     | `/stakerscount-total/{period}`                         |
+| All reward events by type (optional) and network       | `/rewards/{period}/?transaction=BonusReward,Reward`    |
+| Aggregated daily rewards by staker or dapp by period   | `/rewards-aggregated/{address}/{period}`               |
+| Stake amount for one participant                       | `/stake-info/{address}`                                |
+
+## The Indexer
```

Comment:
Add some examples of indexer usage.


Ideally there is a follow up PR with a tutorial (as discussed with the team)